queue_util/consumer.py

```diff
@@ -169,7 +169,7 @@ def run_forever(self, wait_timeout_seconds=None, **kwargs):
                 #
                 break
 
-            except:
+            except Exception:
```

Comment:
Is there much of a difference between `except` and `except Exception`? They'll both 'catch' all exceptions.

Reply:
👋 hey @sujaymansingh - no massive difference, but there are some slight nuances: https://moserei.de/2017/09/20/python-bare-except.html

Reply:
Thanks @Fizzadar, good to know you can now kill these bad boys with a Ctrl-C straight to the heart.

Reply:
👍 That makes sense @Fizzadar , I didn't know about that! (In fact, I would put an explicit `except KeyboardInterrupt` statement and exit from there, something that is probably spread across the code that you guys have to deal with now :))